src/apps/frontend/components/typography/heading-large.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { PropsWithChildren } from 'react';
+
+const HeadingLarge: React.FC<PropsWithChildren> = ({
+  children,
+}) => (
+  <p className="text-3xl font-semibold text-black">
```

Comment:
Why are using `p` for heading and not `h` tags? This is semantically invalid HTML